Network/NetUtils.cpp

```diff
@@ -79,91 +78,6 @@ namespace WPEFramework {
             return (it != interface_descriptions.end()) ? it->second : empty;
         }
 
-        /*
-         * Returns >= 0 on success
-         * Blocking process so run in background thread
-         *
-         * If 'result' is not NULL then it will be set to the return status of the command
-         *
-         * If 'outputfile' is not NULL then it should contain a name for the outpur file. The output of the command will
-         *     be dumped to '/tmp/[outputfile]x' where x is a unique number.
-         *     The filepath will be returned in 'output' and the file should be deleted by the calling process after use.
-         */
-        int NetUtils::execCmd(const char *command, std::string &output, bool *result, const char *outputfile)
-        {
-            std::string commandString;
-            std::string resultFile;
-            char buffer[MAX_OUTPUT_LENGTH];
-            FILE *pipe = NULL;
-            size_t length = 0;
-
-            output.clear();
-
-            commandString.assign(command);
-
-            // If we have an output file then pipe the output from the command to /tmp/outpufile
-            if (outputfile)
-            {
-                getTmpFilename(outputfile, output); //append a count to get a unique filename and return the filename in output
-                commandString += " > ";
-                commandString += output;
-            }
-
-            // If we have requested a result, store the result of the command in /tmp/cmdresult
-            if (result)
-            {
-                getTmpFilename(COMMAND_RESULT_FILE, resultFile); //append a count to get a unique filename
-                commandString += "; echo $? > ";
-                commandString += resultFile;
-            }
-
-            pipe = popen(commandString.c_str(), "r");
```

Comment:
Do we need to remove NetUtils::execCmd? We can just change the popen call within this wrapper API to v_secure_popen so that other modules within Network can continue to use the helper function execCmd.